src/getting-started/tooling/espflash.md

```diff
@@ -1,85 +1,13 @@
 # `espflash`
 
-`espflash` is a serial flasher utility for Espressif SoCs and modules.
-
-The [`espflash`][espflash] repository contains two crates, `cargo-espflash` and `espflash`. For more information on these crates, see the respective sections below.
+[`espflash`][espflash] is a serial flasher utility  designed for Espressif SoCs and modules. It natively supports all chips compatible with `esp-hal`.
 
 [espflash]: https://github.com/esp-rs/espflash
 
-## `espflash`
-
-Provides a standalone command-line application for flashing, monitoring, erasing, and reading memory, along with many other features. For a complete list of available commands, refer to the [Usage section][espflash-usage] of the README.
-
-`espflash` can also be [used as a library][espflash-library], supports a [configuration file][espflash-config], works within [WSL2][espflash-wsl2], and includes support for [`defmt`][espflash-formats].
-
-[espflash-usage]: https://github.com/esp-rs/espflash/tree/main/espflash#usage
-[espflash-config]: https://github.com/esp-rs/espflash/tree/main/espflash#configuration-file
-[espflash-wsl2]: https://github.com/esp-rs/espflash/tree/main/espflash#windows-subsystem-for-linux
-[espflash-library]: https://github.com/esp-rs/espflash/tree/main/espflash#using-espflash-as-a-library
-[espflash-formats]:https://github.com/esp-rs/espflash/tree/main/espflash#logging-format
-
-### Installation
+## Installation
```

Comment:
Maybe we should also mention that binaries are available in gh release and you can use cargo-binstall like we do for cargo-generate.

Also, we should point to the espflash readme, we currently only point to the repo readme, which includes espflash and cargo-espflash, that may be confusing for users as we dont mention cargo-espflash here